InnerEye/ML/deep_learning_config.py

```diff
@@ -222,6 +222,9 @@ class WorkflowParams(param.Parameterized):
     local_weights_path: List[Path] = param.List(default=[], class_=Path,
                                                 doc="A list of checkpoints paths to use for inference, "
                                                     "when the job is running outside Azure.")
+    experiment_name: str = param.String(default="",
```

Comment:
this will mean that there are 2 fields that will auto-generate a commandline argumnet experiment_name. Will this cause problems? Can you double-check?